samples/client/petstore/typescript-nestjs/builds/reservedParamNames/api/default.service.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * Reserved param names
+ * Test reserved param names
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+/* tslint:disable:no-unused-variable member-ordering */
+
+import { Injectable, Optional } from '@nestjs/common';
+import { HttpService } from '@nestjs/axios';
+import { AxiosResponse } from 'axios';
+import { Observable, from, of, switchMap } from 'rxjs';
+import { Configuration } from '../configuration';
+import { COLLECTION_FORMATS } from '../variables';
+
+/**
+ * Request parameters for testReservedParamNames operation in DefaultService.
+ * @export
+ * @interface DefaultServiceTestReservedParamNamesRequest
+ */
+export interface DefaultServiceTestReservedParamNamesRequest {
+    /**
+     * Might conflict with rxjs import
+     * @type {string}
+     * @memberof DefaultServiceTestReservedParamNames
+     */
+    readonly from: string
```

Comment:
I'm not sure to understand what you mean by quoted here

Reply:
Sorry, like this: https://stackoverflow.com/questions/43124403/is-there-a-way-to-escape-and-use-reserved-words-in-typescript-class-definitions

 And an example of the (automatically added) prefix: https://github.com/planet-a-ventures/affinity-node/blob/main/src%2Fv2%2Fgenerated%2Fmodels%2FEmail.ts#L33

You can see that the properties in the typescript generator are also all quoted by default.

Reply:
I've added the quotes.
I saw how sanitized names works with the `reservedWords` Set, I'm not sure `from` should be considered as a reserved word here as it's not a problem anymore after the modifications. But if you confirm you want it added to the `reservedWords` I can add it

Reply:
in general its better to add to the reserved keywords instead of the modifications you suggest, since there are many places where variables are used, so the code change is rather extensive and complicates the templates.

Reply:
Instead of the quotes or even instead of the removal of the destructuration ?

Reply:
once it is a reserved word, the destructure wouldn't be an issue anymore, would it? I am not 100% clear on how the resverved words work, yet, sorry.

Reply:
the reserved keyword would be instead of the quotes, which will prefix the variable with an underscore, e.g. `_from`. however, for the model this makes it differ from what the server actually sends, so probably for the model the quotes are fine. i would  suggest to keep this PR as small as possible and only fix one thing at a time, to make it easier to review

Reply:
From what I understand if we add it to the reserved keywords it will be changed in the model too. Or there is a way to get a non-sanitized name in the model ?

Reply:
you are right, adding to the reserved keywords changes it for the model as well. you could apply the same as in https://github.com/OpenAPITools/openapi-generator/blob/ed211050f1c9d9c45456c006ba1ded30271d1d91/modules/openapi-generator/src/main/resources/typescript-angular/modelGeneric.mustache#L20

Reply:
It should be better now 😃